
int32 trajectory_id
---
bool result

